### ISSUE TITLE
startup error message

### ISSUE THREAD

**Issue by ijgnd** [k-kirill/anki-libunity#1]
The add-on seems to work well for me and looks useful. Thank you very much for creating and sharing this.

During startup I get an error message (I use 2.1.35 on linux and only have your add-on enabled):

```
Caught exception:
Traceback (most recent call last):
  File "aqt/progress.py", line 53, in handler
  File "aqt/main.py", line 143, in on_window_init
  File "aqt/gui_hooks.py", line 1730, in __call__
  File "...../addons21/901237400/__init__.py", line 8, in set_progress
    due_card_count = len(mw.col.find_cards("is:new or is:due"))
AttributeError: 'NoneType' object has no attribute 'find_cards'
```

I removed it with at the beginning of set_progress:

```
    if mw.col is None:
        return
```

My coding skills are also limited.

**Comment by k-kirill** [comment]
You're very welcome and thanks for your feedback!
It's gonna be fun when two people are trying to fix a bug who know nothing about Python :smile: 
Could you please check the following solution first?
Please substitute this hook
```
gui_hooks.main_window_did_init.append(set_progress)
```
by this one:
```
gui_hooks.collection_did_load.append(set_progress)
```
As maybe I used a wrong hook that doesn't always work.

**Comment by ijgnd** [comment]
`gui_hooks.collection_did_load.append(lambda _: set_progress())` seems to work for me. collection_did_load is also one of the hooks that requires an argument.

**Comment by k-kirill** [comment]
Awesome, I've committed the fix with the hook change: 61a7590. Appreciate your help!
P.S. I've re-uploaded the add-on with the new code to https://ankiweb.net/shared/info/901237400.

**Comment by kunpeng9** [comment]
You're very welcome and thanks for your feedback!
It's gonna be fun when two people are trying to fix a bug who know nothing about Python 😄
Could you please check the following solution first?
Please substitute this hook

非常欢迎并感谢您的反馈! 当两个人正在尝试修复一个对 Python 一无所知的 bug 时，这将是一件非常有趣的事情。请您先查看下面的解决方案好吗？请用这个钩子代替

gui_hooks.main_window_did_init.append(set_progress)
by this one:

被这个人:

gui_hooks.collection_did_load.append(set_progress)
As maybe I used a wrong hook that doesn't always work.
这个是啥意思？没有搞明白，是要编辑python文件吗？What does this mean? Do you want to edit the python file?
大概啥时候能更新，我加个任务，等待；About when can update, I add a task, wait;
